js/app.js

```diff
@@ -26,13 +26,14 @@ var DataView = Backbone.View.extend({
     //    });
   },
   render: function() {
+    var resource = this.dataset.toJSON();
```

Comment:
This looks like a formatting change? Complicated by the fact that GitHub says the comment/change is outdated. The resource is needed to be able to put the resource ID in the query text, but AFAIK that wasn't from me.